tests/API/probes/fake_mtab

```diff
@@ -0,0 +1,3 @@
+/dev/mapper/fedora-root / ext4 rw,seclabel,relatime 0 0
```

Comment:
@jan-cerny Can we add more entries into this fake mtab? At least 2 more local filesystems (for example proc, sysfs or tmpfs) and one more remote filesystem?

Reply:
@matusmarhefka done

Reply:
@jan-cerny Great, thank you.